all/build.gradle

```diff
@@ -18,6 +18,7 @@ def subprojects = [
     project(':grpc-protobuf-lite'),
     project(':grpc-stub'),
     project(':grpc-testing'),
+    project(':grpc-xds'),
```

Comment:
Is this collection used for both javadoc and jacoco?

Reply:
Yes. It also adds a dependency in the pom to grpc-all.